libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/factory/RouteProgressFactory.kt

```diff
@@ -55,7 +56,8 @@ object RouteProgressFactory {
         fractionTraveled: Float,
         remainingWaypoints: Int,
         upcomingRoadObjects: List<UpcomingRoadObject>,
-        stale: Boolean
+        stale: Boolean,
+        layer: Int?,
```

Comment:
Could we call the value itself something like `zLevel`? `Layer` is a very overloaded term in a Mapbox ecosystem.

Reply:
I think we should also expose a getter for this value through `MapboxNavigation`.

Typically when a route request is made, it's not done from the `RouteProgress` callback, it's trigger by some user interaction, so developers will come looking for this value and will have to cache it themselves anyway.